test/clj/rems/test_browser.clj

```diff
@@ -105,7 +109,9 @@
 (defn slurp-fields [selector]
   (->> (for [row (btu/query-all [selector {:fn/has-class :form-group}])
              :let [k (btu/get-element-text-el (btu/child row {:tag :label}))
-                   [value-el] (btu/children row {:css ".form-control"})]
+                   value-els (concat (btu/children row {:css ".form-control"})
+                                     (btu/children row {:css ".dropdown-container"}))]
+             value-el value-els
```

Comment:
I think this is a bit confusing now. I'd either
- get rid of the `value-els` let (and inline it), or
- use something like `:let [value-el (find-first some? (concat ...))]`

Reply:
I'll try adding some helpers to clarify. I don't like either of these.